Added styling options and alwaysOn option

BeFiveINFO opened the pull request:
Please ignore README.md, as I modified the file for leaving a brief summary memo of changes.

Changes made to the  Phasetips.js are:

1. Flexible pinpoint Text Style Options
2. roundedCornersRadius with fallback (older phaser might not have
drawRoundRect)
3. alwaysOn to skip mouse hover events
4. simulateOnHoverOver / simulateOnHoverOut methods to simulate the
hover event (to make use of the fadeout / fadein).

Thank you for your consideration.

BeFiveINFO committed:
added few more options

few more options for show all (shows always while ignoring hover), font styles, rounded corners (requires the later version of phaser JS. works fine with Phaser v2.6.2).

BeFiveINFO committed:
Update README.md

BeFiveINFO committed:
Update README.md

BeFiveINFO committed:
Update README.md

BeFiveINFO committed:
Added more styling options

1. Flexible pinpoint Text Style Options
2. roundedCornersRadius with fallback (older phaser might not have
drawRoundRect)
3. alwaysOn to skip mouse hover events
4. simulateOnHoverOver / simulateOnHoverOut methods to simulate the
hover event (to make use of the fadeout / fadein).

samme commented on README.md:
```diff
@@ -1,92 +1,40 @@
+var tooltip_enter_name = new Phasetips(Game, {
+	targetObject: Register.spriteInstances.name_input_text_field_backpanel,
+	context: "Please enter your nickname to start game.",
+	fontSize: "24px Josefin Sans",
```
Is `fontSize` a size (number) or font declaration (string)?

netgfx replied:
according to this `_fontSize = _options.fontSize || 12;` its a number. to use string declaration you need to use `font` instead

BeFiveINFO replied:
Hello, Thank you for asking. fontSize is font declaration (string). (I was confused with font and fontSize.)

I am leaving a memo for us and others now and in the future just in case: http://phaser.io/docs/2.6.2/Phaser.Text.html

The added options for the text style  is according to the specification listed there in "Parameters". You can find font property in the style parameter category. The type specified is "string" and accepts a value like 'bold 20pt Arial' (namely "The style and size of the font.")

BeFiveINFO replied:
added font option

netgfx commented:
@BeFiveINFO Please modify READ.me to showcase the new changes so I can merge them automatically.
Also, change the `fontSize` to a single number on your example

Thanks!

Commit:
Added font option to text style

"font" option is added to accept the style and size setting of the font
in string value all at once.

BeFiveINFO committed:
Added additional options, heading titles by group

BeFiveINFO committed:
textStyle default value styling added

Commit:
Default values changed according to the documentation

Commit:
removed unnecessary check up for drawRoundedRect

drawRoundedRect has already been available in Phaser JS version 2.2.2.

BeFiveINFO committed:
roundedCornersRadius option added

BeFiveINFO committed:
spell mistake corrected

BeFiveINFO committed:
missing colons added

BeFiveINFO committed:
additional desc. on textStyle option behavior

BeFiveINFO commented:
Added the new options to the README along with group styling for the options. Correction added to the main source code including that of font option.

Thank you for a review once again!

BeFiveINFO committed:
position option regrouped

BeFiveINFO commented:
please hold on. I found something wrong

Commit:
wrong conditional statement broke the plugin

Commit:
added rounded corners example

BeFiveINFO commented:
Added examples and fixed a bug with the rounded corners option. Now ready for a review. Thank you!
